dtfs-central-api/src/repositories/utilisation-reports-repo/utilisation-report-sql.repo.ts

```diff
@@ -42,7 +42,21 @@ export const UtilisationReportRepo = SqlDbDataSource.getRepository(UtilisationRe
       findByOptionsWhere.status = Not('REPORT_NOT_RECEIVED');
     }
 
-    return await this.findBy(findByOptionsWhere);
+    return await this.find({
+      where: findByOptionsWhere,
+      order: {
+        reportPeriod: {
+          start: {
+            year: 'ASC',
+            month: 'ASC',
+          },
+          end: {
+            year: 'ASC',
+            month: 'ASC',
+          },
+        },
```

Comment:
How does the order of application the different field orderings get determined from this?

Reply:
The sort order is determined from the order of this list, so order is:
- Start year
- Start month
- End year
- End month

Reply:
Nice thanks!